elements/nuxeo-csv-export/nuxeo-csv-export-button.html

```diff
@@ -78,35 +83,43 @@
           type: Boolean,
           value: false,
         },
+
+        /**
+         * Current action status.
+         */
+        status: {
+          type: Object,
+          notify: true,
+        },
+
         _label: {
           type: String,
           computed: '_computeLabel(i18n)',
-        },
-        _lastStatus: Object
+        }
       },
 
-      execute: function() {
-        this.$.resource.path = this.provider.$.nxResource.path.replace(/\/execute$/, '/bulk/csvExport');
-        this.$.resource.params = this.provider.$.nxResource.params;
-        this.$.resource.data = {};
-        var schemas = this.schemas != null ? this.schemas : this.provider.$.nxResource.schemas;
-        if (schemas) {
-          this.$.resource.data.schemas = schemas.split(',').map(function(schema) {
-            return schema.trim();
-          });
+      _params: function() {
+        var actionParams = {};
+        if (this.schemas) {
+          actionParams.schemas = this.schemas.split(',').map(function(s) { return s.trim(); });
```

Comment:
Need to set the provider's schemas instead if no schemas were passed to the operation. Something alongside these lines (and ditching the if):
```suggestion
          actionParams.schemas = this.schemas ? this.schemas.split(',').map(function(s) { return s.trim(); }) : this.provider.schemas;
```

Reply:
Done 👍 Moved parameter building to execution time just to be safe (as it was before).